src/test/scala/mdf/macrolib/ConfReaderSpec.scala

```diff
@@ -1,10 +1,9 @@
-package mdf.macrolib.test
```

Comment:
Is there a reason to include test classes into the main package?

Reply:
@edwardcwang It's a pretty standard layout and you get access to the classes in your main without needing imports. It's not critical but just keeps things simple